build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -1,14 +1,10 @@
 <!--
-  1. Change "category" below to one of:
-       ads-analytics
-       dynamic-content
-       layout
-       media
-       presentation
-       social
-  2. Remove any of the "formats" that don't apply.
-     You can also add the "ads" and "stories" formats if they apply.
-  3. And remove this comment! (no empty lines before "---")
+  All documentation starts with frontmatter. Front matter organizes documentation on amp.dev
+  and improves SEO.
+  Include the relevant category(ies): ads-analytics, dynamic-content, layout, media,
+  presentation, social, personalization
+  List applicable format(s): websites, ads, stories, email
+  And remove this comment! (no empty lines before "---")
```

Comment:
Would this be easier to read as a bulleted list?

Reply:
Thanks Raghu! Updated as requested